docs/C_API.md

```diff
@@ -28,6 +28,8 @@ chooses to override this by setting ```session_state.use_env_allocators``` to "0
    * Set ```session.use_env_allocators``` to "1" for each session that wants to use the env registered allocators.
    * See test ```TestSharedAllocatorUsingCreateAndRegisterAllocator``` in
      onnxruntime/test/shared_lib/test_inference.cc for an example.
+* **Share initializer(s) between sessions:** This feature allows a user to share the same instance of an initializer across
+multiple sessions. Use the ```AddInitializer``` API to register a pre-allocated initializer with the session before calling ```CreateSession```.
 
```

Comment:
Is there somewhere we could add actual example usage code? 

Do you add the initializer to 'the session' or to shared SessionOptions? 

Would also be good to say when/why you'd want to have a shared instance of an initializer.